example/package.json

```diff
@@ -7,7 +7,7 @@
   },
   "dependencies": {
     "react": "^15.1.0",
-    "react-native": "^0.27.0-rc2",
-    "react-native-calendar": "github:christopherdro/react-native-calendar#revamp"
+    "react-native": "^0.27.0",
+    "react-native-calendar": "file:../"
```

Comment:
Updated `react-native` to latest and also pulling `react-native-calendar` from parent directory, makes things easier to debug. When making changes in parent be sure to `npm un react-native-calendar && npm i` to remove previous version and to pull in fresh from parent directory.